### PR TITLE
[WIP] Migrate to MySQL

### LINKED ISSUE #150: Sqlite to MySQL

### PR DESCRIPTION
**Pull request by dimkarakostas**
Fix #150

### CODE REVIEW

**Comment by dimkarakostas** [comment]
@dionyziz a little help with the failing build?

**Comment by dionyziz** [comment]
Use the username travis [and a blank password](https://docs.travis-ci.com/user/database-setup/#MySQL) and make sure you have started mysql using the `mysql` service in `.travis.yml`.

File: backend/setup_backend.sh [hunk 1]
```diff
@@ -2,12 +2,8 @@
 
 BASEDIR=$(dirname "$0")
 
-echo '[-] Removing old database, if exists...'
-rm -f $BASEDIR/db.sqlite3
-if test -e "$BASEDIR/db.sqlite3"; then
-    echo "[!] Database could not be deleted. Check permissions on $BASEDIR/db.sqlite3?";
-    exit;
-fi
+echo "[*] Setup of the database requires access to MySQL server as root."
```
Comment (dionyziz) [inline backend/setup_backend.sh]: This seems somewhat unreasonable. If the user can create a MySQL user for us with the appropriate privileges, we should be able to continue.

Comment (dimkarakostas) [inline backend/setup_backend.sh]: Sure he can, this script just makes the setup fully automated and saves the user the trouble. Do you think we should remove this and give the user the necessary steps in the installation instructions instead?

Comment (dionyziz) [inline backend/setup_backend.sh]: I'm saying give the user the ability to pass a username, database name, and password to this script if they want to. Also store these details in the conf file.

File: backend/init.sql [hunk 2]
```diff
@@ -0,0 +1,4 @@
+DROP DATABASE IF EXISTS rupture;
+CREATE DATABASE rupture CHARACTER SET UTF8;
+GRANT ALL ON rupture.* TO 'rupture'@'localhost' IDENTIFIED BY 'rupture';
```
Comment (dionyziz) [inline backend/init.sql]: Randomize this password in the calling script and then store it. Default passwords are dangerous.

**Comment by dionyziz** [comment]
Packaging will be much easier with SQLite. Are we sure we want to migrate to MySQL and why?

**Pull request closed without merging**